backend/db/queries/orderQueries.js

```diff
@@ -100,3 +100,10 @@ module.exports = {
   getAllOrderForSingleUser,
   allOrdersForGrandma
 };
+
+//const orderSummary = (req, res, next) => {
+
+// const user_id = parseInt(req.params.user_id;
+//
+// db.any(
+//   "SELECT users.id AS user_id, first_name, last_name, email, dishes.user_id AS grandma_id, phone_number, isGrandma, building_number, address, zip_code, dishes.id AS dish_id, name, price, timeframe, orders.id AS order_id, isCompleted  FROM orders JOIN users ON orders.user_id = users.id JOIN dishes ON dishes.id = orders.dish_id WHERE users.id=$1", [user_id]
```

Comment:
why are we committing commented out code?

Reply:
remember. ideally a PR should only contain code that does one thing. for example Adds/removes a piece of functionality, or fixes a bug, or even adds some comments or cleans up some SCOPED piece of code.